src/views/DataExploration/components/PageContent/tabs/Summary/AgeAtDiagnosisGraphCard/index.tsx

```diff
@@ -0,0 +1,162 @@
+import { RawAggregation } from 'graphql/models';
+import { toChartData } from 'utils/charts';
+import BarChart from 'components/uiKit/charts/Bar';
+import GridCard from '@ferlab/ui/core/view/v2/GridCard';
+import { truncateString } from 'utils/string';
+import { INDEXES } from 'graphql/constants';
+import { useHistory } from 'react-router-dom';
+import { ArrangerValues } from '@ferlab/ui/core/data/arranger/formatting';
+import { isEmpty } from 'lodash';
+import Empty from '@ferlab/ui/core/components/Empty';
+import { AGE_AT_DIAGNOSIS_QUERY } from 'graphql/summary/queries';
+import { ARRANGER_API_PROJECT_URL } from 'provider/ApolloProvider';
+import useApi from 'hooks/useApi';
+import useParticipantResolvedSqon from 'graphql/participants/useParticipantResolvedSqon';
+import CardHeader from 'views/Dashboard/components/CardHeader';
+import intl from 'react-intl-universal';
+import { DATA_EXPLORATION_QB_ID } from 'views/DataExploration/utils/constant';
+import { updateActiveQueryField } from '@ferlab/ui/core/components/QueryBuilder/utils/useQueryBuilderState';
+import { RangeOperators } from '@ferlab/ui/core/data/sqon/operators';
+
+interface OwnProps {
+  id: string;
+  className?: string;
+}
+
+type DiagnosisQueryResult = {
+  data: {
```

Comment:
if possible add as much properties as possible. You could leave `[key: string]` too for unknown properties